wifiphisher/pywifiphisher.py

```diff
@@ -158,6 +170,21 @@ def parse_args():
         help="Force the usage of hostapd installed in the system",
         action='store_true')
 
+    parser.add_argument("-dP",
+                        "--data_path",
+                        help="Search for phishing pages in this location")
+    parser.add_argument(
+        "-dns",
+        "--dns_path",
+        help="the full path+filename for the dnsmasq.conf file",
```

Comment:
`-dC` or `--dnsmasq-conf`.

And then `Determine the full path of a custom dnmasq.conf file`.